.travis.yml

```diff
@@ -1,16 +1,34 @@
 language: rust
+
+before-install:
+  - curl --location https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
+
 rust:
   - nightly
-cache: cargo
```

Comment:
Why are we removing this? (This potentially makes the build quite a lot slower)

Reply:
Nick recommended it for the subgraph project sometime ago, I forgot the exact reason. I will put it back in. I think it has no effect in this case.